src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const employees = [ ...list.children ];
+
+function normalizeSalary(salary) {
+  return +salary.replace('$', '').replace(',', '');
```

Comment:
what if there's more than 1 `,`? try to find a better solution
![image](https://user-images.githubusercontent.com/37161315/174807943-3f9ad600-aca0-4cbd-b151-1386ca0f07a1.png)